gr-uhd/lib/rfnoc_rx_radio_impl.cc

```diff
@@ -31,6 +34,14 @@ rfnoc_rx_radio::sptr rfnoc_rx_radio::make(rfnoc_graph::sptr graph,
 rfnoc_rx_radio_impl::rfnoc_rx_radio_impl(::uhd::rfnoc::noc_block_base::sptr block_ref)
     : rfnoc_block(block_ref), d_radio_ref(get_block_ref<::uhd::rfnoc::radio_control>())
 {
+#define REGISTER_CMD_HANDLER(key, _handler)                                   \
+    register_msg_cmd_handler(                                                 \
+        key, [this](const pmt::pmt_t& var, int chan, const pmt::pmt_t& msg) { \
+            this->_handler(var, chan, msg);                                   \
+        })
+    // Register default command handlers:
+    REGISTER_CMD_HANDLER(pmt::symbol_to_string(cmd_stream_cmd_key()),
+                         _cmd_handler_stream_cmd);
```

Comment:
Hm, my two cents:

not a fan of `#defines`, but OK. However, this is used only once, so the result of actually writing it out would be easier to read.

Reply:
I'll be adding more, like here: https://github.com/gnuradio/gnuradio/blob/ff6d992f9a7877f431c62854112114580ebeb9c5/gr-uhd/lib/usrp_block_impl.cc#L158-L177

...just not in this PR.